printing_auto_base/tests/test_printing_auto_base.py

```diff
@@ -4,19 +4,25 @@
 
 from unittest import mock
 
+from odoo_test_helper import FakeModelLoader
+
 from odoo.exceptions import UserError, ValidationError
 
-from .common import PrintingPrinter, TestPrintingAutoCommon, print_document
-from .model_test import PrintingAutoTester, PrintingAutoTesterChild, setup_test_model
+from odoo.addons.base_report_to_printer.models.printing_printer import PrintingPrinter
+
+from .common import TestPrintingAutoCommon, print_document
+from .model_test import PrintingAutoTester, PrintingAutoTesterChild
```

Comment:
You should import the models only inside the setup and only after the backup https://github.com/OCA/search-engine/blob/16.0/connector_search_engine/tests/test_all.py#L24